modules/olm-installing-opm.adoc

```diff
@@ -84,7 +84,7 @@ C:\> oc image extract {registry-image} \
 ifndef::openshift-origin[]
     -a ${REG_CREDS} \
 endif::[]
-    --path /usr/bin/registry/windows-amd64-opm:. \
+    --path /usr/bin/opm:. \
```

Comment:
Same question for @zhouying7780 if the binary needs to stay `windows-amd64-opm` or if it should just be `opm` for windows.